plugins/zsh-syntax-highlighting/index.ts

```diff
@@ -19,6 +21,24 @@ const plugin: Fig.Plugin = {
     origin: "github",
     sourceFiles: "zsh-syntax-highlighting.zsh",
   },
+  configuration: [
+    {
+      displayName: "Max Length",
+      type: "environmentVariable",
+      description: "Prevent highlighting for commands greater than this length",
+      interface: "text",
+      default: 512,
+      environmentVariable: "ZSH_HIGHLIGHT_MAXLENGTH",
+    },
+    {
+      displayName: "Highlighters",
+      type: "environmentVariable",
+      description: "Choose which higlighters you want to use (default is all of them)",
+      interface: "text",
+      default: "(main brackets pattern cursor)",
```

Comment:
```suggestion
      interface: "multiselect",
      default: ["main", "brackets", "pattern", "cursor"],
      options: ["main", "brackets", "pattern", "cursor", ...otherOptions],
```

Seems like this would work better as a `multiselect` since it is a list variable. We should probably add an interface for multiselects where the user can add other string options as well (something between the current `text` and `multiselect` interfaces)